MAINTAINERS.md

```diff
@@ -0,0 +1,90 @@
+# Maintainers
+
+The current technical steering committee (TSC) are also maintainers of this repo. More information can be found [here](https://github.com/hiero-ledger/tsc).
+
+## Active Maintainers
+
+| Name            |	GitHub ID	                                   | GitHub Scope |	LFID  |	Discord ID  |	Email    | 
+|-----------------|----------------------------------------------|--------------|-------|-------------|----------|
+|                 | [@rwalworth](https://github.com/rwalworth)   | MAINTAIN     |       |             |          |
+|                 | [@SimiHunjan](https://github.com/SimiHunjan) | MAINTAIN     |       |             |          |
+
+## Emeritus Maintainers
+
+NONE
+
+## The Duties of a Maintainer
+
+Maintainers are expected to perform the following duties for this repository.
+The duties are listed in more or less priority order:
+
+- Review, respond, and act on any security vulnerabilities reported against the repository.
+- Review, provide feedback on, and merge or reject GitHub Pull Requests from Contributors.
+- Review, triage, comment on, and close GitHub Issues submitted by Contributors.
+- When appropriate, lead/facilitate architectural discussions in the community.
+- When appropriate, lead/facilitate the creation of a product roadmap.
+- Create, clarify, and label issues to be worked on by Contributors.
+- Ensure that there is a well defined (and ideally automated) product test and release pipeline,
+  including the publication of release artifacts.
+- When appropriate, execute the product release process.
+- Maintain the repository CONTRIBUTING.md file and getting started documents to give guidance and
+  encouragement to those wanting to contribute to the product, and those wanting to become maintainers.
+- Contribute to the product via GitHub Pull Requests.
+- Monitor requests from the LF Decentralized Trust Technical Advisory Council about the contents
+  and management of LFDT repositories, such as branch handling, required files in repositories and so on.
+- Contribute to the LFDT Project's Quarterly Report.
+
+## Becoming a Maintainer
+
+This community welcomes contributions.
+Interested contributors are encouraged to progress to become maintainers.
+To become a maintainer the following steps occur, roughly in order.
+
+- The proposed maintainer establishes their reputation in the community,
+  including authoring five (5) significant merged pull requests, and expresses
+  an interest in becoming a maintainer for the repository.
```

Comment:
I think for us, we distinguish between committers and maintainers, whereas LFDT may not in general. I think the bar to becoming a maintainer is far higher than 5 significant merged pull requests, although for a committer this may be about right. I think we need to revisit this language. Maintainers have voting rights, and that should come with a long history of active participation.